imports/components/sectionHeader/index.js

```diff
@@ -0,0 +1,22 @@
+
+const halvesClasses = "one-half floating display-inline-block soft-half";
+
+type ISectionHeader = {
+  title?: string,
+  link?: Object, // React component
+};
+
+const SectionHeader = ({ title, link }: ISectionHeader) =>
+  <div className="background--light-secondary one-whole">
```

Comment:
@JakeDawkins let's leave the coloring to the parent component, so you can remove the background color here